app/webpacker/components/ContactGearOrderPage/GearOrderForm.jsx

```diff
@@ -0,0 +1,170 @@
+/* eslint-disable max-len */
+import _ from 'lodash';
+import React, { useEffect, useState } from 'react';
+import {
+  Form, Header, HeaderSubheader, Input, Table,
+  TableHeader,
+  TableHeaderCell,
+  TableRow,
+} from 'semantic-ui-react';
+
+export default function GearOrderForm({ equipments }) {
+  const [orderDetails, setOrderDetails] = useState(equipments.map(() => ({
+    quantity: 0,
+    price: 0,
+  })));
+  const [totalPrice, setTotalPrice] = useState(0);
```

Comment:
Does this work? Direct calculation is better than a state and an effect.
```suggestion
  const totalPrice = _.sumBy(orderDetails, 'price');
```